network/p2p/dns/resolver_test.go

```diff
@@ -76,33 +75,35 @@ func TestResolver_CacheExpiry(t *testing.T) {
 		metrics.NewNoopCollector(),
 		dnsCache,
 		WithBasicResolver(&basicResolver),
-		WithTTL(1*time.Second)) // cache timeout set to 1 seconds for this test
+		WithTTL(3*time.Second)) // cache timeout set to 3 seconds for this test
 
 	cancelCtx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	ctx, _ := irrecoverable.WithSignaler(cancelCtx)
 	resolver.Start(ctx)
 	unittest.RequireCloseBefore(t, resolver.Ready(), 100*time.Millisecond, "could not start dns resolver on time")
 
-	size := 10 // we have 10 txt and 10 ip lookup test cases
-	times := 5 // each domain is queried for resolution 5 times
+	size := 5  // we have 5 txt and 5 ip lookup test cases
+	times := 3 // each domain is queried for resolution 3 times
 	txtTestCases := testnetwork.TxtLookupFixture(size)
 	ipTestCase := testnetwork.IpLookupFixture(size)
 
 	// each domain gets resolved through underlying resolver twice: once initially, and once after expiry.
 	resolverWG := mockBasicResolverForDomains(t, &basicResolver, ipTestCase, txtTestCases, happyPath, 2)
 
+	// queries (5 + 5) cases * 3 = 100 queries.
```

Comment:
is this math correct? (5+5) * 3 = 30 queries

Reply:
Thanks, fixed it here: https://github.com/onflow/flow-go/pull/2670/commits/ef0832baecc78883ec4000be1f746dfa5b25b9cc